client/my-sites/posts/post-controls.jsx

```diff
@@ -45,7 +46,7 @@ const getAvailableControls = props => {
 	// Main Controls (not behind ... more link)
 	if ( userCan( 'edit_post', post ) ) {
 		controls.main.push( {
-			className: 'edit',
+			className: 'edit' + ( current === 'edit' ? ' is-active' : '' ),
```

Comment:
I believe we are taking advantage of `classNames` npm module and it it working quite nicely
https://www.npmjs.com/package/classnames